enterprise-suite/templates/alertmanager.yaml

```diff
@@ -3,6 +3,8 @@ kind: PersistentVolumeClaim
 apiVersion: v1
 metadata:
   name: alertmanager-storage
+  annotations:
+    "helm.sh/resource-policy": keep
```

Comment:
is this annotation necessary when installing without helm

Reply:
Not necessary but it doesn't hurt. I don't think we should make a flag for it.